api/src/main/java/edu/cornell/mannlib/vitro/webapp/utils/configuration/PropertyType.java

```diff
@@ -60,10 +57,23 @@ public PropertyStatement buildPropertyStatement(Statement s) {
 
 		@Override
 		protected PropertyMethod buildPropertyMethod(Method method,
-				Property annotation) {
+													 Property annotation) {
```

Comment:
Some odd formatting here and other places.

Reply:
Resolved, thanks.